analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -90,7 +85,24 @@ def parse_args() -> argparse.Namespace:
         "--hosts",
         help="A comma-separated list of worker node hostnames. Only required if --scheduler=dask-ssh, ignored otherwise.",
     )
-    p.add_argument("-v", "--verbose", help="Turn on verbose execution logs.", action="store_true")
+    p.add_argument(
+        "-v",
+        "--verbose",
+        help="Turn on verbose execution logs.",
+        action="store_true",
+    )
+
+    p.add_argument(
+        "--statistical-validation",
+        help="Run only statistical validation part of the analysis.",
+        action="store_true",
+    )
```

Comment:
If we put it like this
```
p.add_argument(
        "--statistical-validation",
        help = argparse.SUPPRESS,
        action="store_true",
    )
```
So it will not be shown when running `python analysis.py --help`?

The reason, why I would like to keep it: now GitHub Actions are divided into first histogram creation using parameter `--no-fitting`, and the second action is run with `--statistical-validation`. It provides an opportunity to see, where exactly the problem appears, in the data obtaining part of the fitting part. And also save time for the second action.

Reply:
After discussion, I see your point and I agree 👍